environment/visibility.csv

```diff
@@ -123,6 +123,7 @@ OHIN,"Excellent, no obstructions",2008-09-03T00:00:00Z,9999-01-01T00:00:00Z
 OKOH,ok,2009-11-16T00:00:00Z,9999-01-01T00:00:00Z
 OPTK,suitable visibility Inc. Dec. 7 137 tree 4 160-140 5 125-110 trees 4 355 tree,2009-08-20T00:00:00Z,9999-01-01T00:00:00Z
 OROA,excellent,2010-11-08T00:00:00Z,9999-01-01T00:00:00Z
+ORRG,excellent,2024-12-05T00:00:00Z,9999-01-01T00:00:00Z
```

Comment:
Hey @EvanCGNS ! I would suggest we change it to something like: "There are bushes in the West, the North West, the North, the North East at the distance of 7-15 meters.", because the conditions are not so perfect.

Reply:
Will change it, cheers